Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -1090,6 +1090,9 @@ struct molops_wrapper {
         .def_readwrite("removeNonimplicit",
                        &MolOps::RemoveHsParameters::removeNonimplicit,
                        "DEPRECATED")
+        .def_readwrite("removeHydrids",
```

Comment:
hydrides is mis-spelled
```suggestion
        .def_readwrite("removeHydrides",
```